ui/desktop/src/components/ChatInput.tsx

```diff
@@ -425,6 +480,38 @@ export default function ChatInput({
   };
 
   const handleKeyDown = (evt: React.KeyboardEvent<HTMLTextAreaElement>) => {
+    // If mention popover is open, handle arrow keys and enter
+    if (mentionPopover.isOpen && mentionPopoverRef.current) {
+      if (evt.key === 'ArrowDown') {
```

Comment:
this could be simplified by using a case statement and extracting an updateSelectedIndex function